cmd/zoekt-webserver/main.go

```diff
@@ -117,7 +116,7 @@ func writeTemplates(dir string) error {
 }
 
 func main() {
-	logDir := flag.String("log_dir", "", "log to this directory rather than stderr.")
```

Comment:
I didn't see any place this was used but it might cause some config to break

Reply:
blocking: This breaks the command line API, which will break OSS users of zoekt.

The root cause of this is that dd-trace-go is importing glog, my least favourite go logging library since it does things like this. https://sourcegraph.com/search?q=context%3Aglobal+repo%3Adeps%28%5Egithub%5C.com%2Fsourcegraph%2Fzoekt%24%40dax%2Fadd_dd_tracer%29+flag+log_dir&patternType=regexp

unfortunetly you need to remove this line then after command line parsing use `flag.Lookup` to work around this bad pkg. Then we need a test to ensure that glog is indeed registering the flag so that if the deps change we don't break our behaviour.